src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -104,8 +137,94 @@ def load_pretrained_quantization(model: Module, model_name_or_path: str):
             )
 
 
+def process_transforms_config(
```

Comment:
I missed that this is actually modifying `model` registering transforms to it. The name `process_transforms_config` is pretty innocuous for something like this, given it's modifying `model` significantly. Maybe `add_transforms_to_model` or something more explicit?